classes/ImageManager.php

```diff
@@ -688,6 +699,14 @@ public static function write($type, $resource, $filename)
 
                 break;
 
+            case 'avif':
+                if (function_exists('imageavif')) {
+                    $quality = ($avifWebpQuality === false ? 80 : $avifWebpQuality);
+                    $success = imageavif($resource, $filename, $quality);
```

Comment:
```suggestion
                    // @phpstan-ignore-next-line
                    $success = imageavif($resource, $filename, $quality);
```
Seems like PHPStan is not happy about this, other lines were ignored I believe it's mostly related to recent PHP versions where resource has been replaced with GdImage or something similar